app/internal/wm/os_android.go

```diff
@@ -728,6 +753,26 @@ func setCursor(env *C.JNIEnv, view C.jobject, name pointer.CursorName) {
 	callVoidMethod(env, view, gioView.setCursor, jvalue(curID))
 }
 
+func setStatusColor(env *C.JNIEnv, view C.jobject, color color.NRGBA) {
+	callVoidMethod(env, view, gioView.setStatusColor,
+		jvalue(uint32(color.A)<<24|uint32(color.R)<<16|uint32(color.G)<<8|uint32(color.B)),
+		jvalue(luminance(color)),
+	)
+}
+
+func setNavigationColor(env *C.JNIEnv, view C.jobject, color color.NRGBA) {
+	callVoidMethod(env, view, gioView.setNavigationColor,
+		jvalue(uint32(color.A)<<24|uint32(color.R)<<16|uint32(color.G)<<8|uint32(color.B)),
+		jvalue(luminance(color)),
+	)
+}
+
+func luminance(color color.NRGBA) int {
+	// Relative Luminance.
+	// https://en.wikipedia.org/wiki/Relative_luminance.
+	return int((0.2126 * float64(color.R)) + (0.7152 * float64(color.G)) + (0.0722 * float64(color.B)))
```

Comment:
Add a +0.5 to round correctly?

Reply:
Oh, and internal/f32color has a RGBA.Luminance method. Use that instead of duplicating the computation.